scio-core/src/main/scala/com/spotify/scio/testing/TestDataManager.scala

```diff
@@ -112,6 +113,8 @@ case class BigQueryIO(tableSpecOrQuery: String) extends TestIO[TableRow](tableSp
 case class DatastoreIO(projectId: String, query: Query = null, namespace: String = null)
   extends TestIO[Entity](s"$projectId\t$query\t$namespace")
 
+case class ProtobufIO[T](path: String)(implicit ev: T <:< Message) extends TestIO(path)
```

Comment:
It doesn't, `ScioContext#protobufFile` is `def protobufFile[T: ClassTag](path: String)(implicit ev: T <:< Message)` so `T` alone doesn't have to be a `Message` (at least according to this API) as long as there is evidence in scope.